src/test/java/com/exasol/containers/ExasolContainerVersionSpecificTest.java

```diff
@@ -22,11 +20,10 @@
 
     @Test
     void testContainer62x() {
-
-        ExasolContainer<? extends ExasolContainer<?>> containerV62 = new ExasolContainer<>("6.2.7-d1", false);
-        var exception = assertThrows(ContainerLaunchException.class, containerV62::start);
-        assertThat(exception.getMessage(), containsString("E-ETC-13"));
-
+        try (ExasolContainer<? extends ExasolContainer<?>> containerV62 = new ExasolContainer<>("6.2.17-d1", false)) {
```

Comment:
Funny, I just fixed the same thing in my branch. :-)

Reply:
😆